src/mesh/Draco_Mesh.cc

```diff
@@ -1,17 +1,17 @@
-//----------------------------------*-C++-*----------------------------------//
+//-----------------------------------*-C++-*----------------------------------//
 /*!
  * \file   mesh/Draco_Mesh.cc
  * \author Ryan Wollaeger <wollaeger@lanl.gov>
  * \date   Thursday, Jun 07, 2018, 15:38 pm
  * \brief  Draco_Mesh class implementation file.
  * \note   Copyright (C) 2018-2019 Triad National Security, LLC.
  *         All rights reserved. */
-//---------------------------------------------------------------------------//
+//----------------------------------------------------------------------------//
 
 #include "Draco_Mesh.hh"
 #include "ds++/Assert.hh"
 #include <algorithm>
-#include <iostream>
+// #include <iostream>
```

Comment:
If this isn't needed, go ahead and delete this line.

Reply:
Thank you for catching this - it has been removed.